### DISCUSSION TITLE
Make Taskbar right-click menu match the start-menu right-click menu.

### DISCUSSION THREAD

**Discussion by yodaluca23** [valinet/ExplorerPatcher#501] [General]
Make TaskBar right-click menu match the start-menu right-click menu. The text in the taskbar right-click is smaller than the start right-click menu. please make it match up.

**Comment by valinet** [comment]
Make a more detailed report. You mean the Windows 11 or 10 taskbar? If 10, it’s the same, if 11, it can’t be easily done, the new taskbar is as “open” as North Korea, despite what Microsoft says.

Anyway, hardly an issue imo, this should have been posted to discussions.
